README.md

```diff
@@ -7,4 +7,15 @@ Run the command `SET ASPNETCORE_Environment=Development` in your terminal.
 To run the app run `dotnet run` in the root of the repository.
 To run the UI tests navigate ClientApp directory and run `npm test`.
 
+If you get an error along the lines of `'rimraf' is not recognized`, navigate to the `ClientApp` folder and run an `npm install`.
+
+# Preparing the database
+Create a `.env` file in the root directory, containing
+```
+DATABASE_CONNECTION_STRING=Server=localhost;port=5432;Database=WhaleSpottingDb;user id=username;password=secret;
```

Comment:
Upon doing the deployment yesterday we'll need this to be called 'DATABASE_URL' for it to work :)